docs/02-app/01-building-your-application/01-routing/10-route-handlers.mdx

```diff
@@ -604,6 +604,46 @@ export async function GET(request) {
 }
 ```
 
+### Webhooks
+
+You can use a Route Handler to receive webhooks from third-party services:
+
+```ts filename="app/api/route.ts" switcher
+export async function POST(request: Request) {
+  try {
+    const text = await request.text()
+    // Process the webhook payload
+  } catch (error) {
+    return new Response(`Webhook error: ${error.message}`, {
```

Comment:
Hello 😶‍🌫️. I accidentally looked in and came across something painfully familiar.

"'error' is of type 'unknown'"

But I'm not sure if it's too much to add a check in the doc like `if (error instanceof Error)`, but on the other hand, it seems like the right thing to do, as if the example shouldn't include an error.